taxonomies/ingredients.txt

```diff
@@ -39633,6 +39912,76 @@ ro:pudră de căpșuni
 fi:mansikoista valmistettu mehu ja viinirypäleet
 
 
+# description:en:Fragaria vesca, commonly called WILD STRAWBERRY, woodland strawberry, Alpine strawberry, Carpathian strawberry, European strawberry, or fraisier des bois, is a perennial herbaceous plant in the rose family that grows naturally throughout much of the Northern Hemisphere, and that produces edible fruits.
+
+<en:fruit
+<fr:fruit rouge
+en:wild strawberry, woodland strawberry, alpine strawberry, carpathian strawberry, fragaria vesca
+ar:توت الأرض المزروع
+az:meşə çiyələyi
+ba:ҡайын еләге
+be:суніцы лясныя
+bg:горска ягода
+ca:maduixera
+cs:jahodník obecný
+da:skovjordbær
+de:wald-erdbeere
+el:χαμοκέρασο
+eo:arbara fragujo
+es:fragaria vesca
+et:metsmaasikas
+eu:marrubi arrunt
+fa:توت‌فرنگی وحشی
+fi:ahomansikka, metsämansikka
+fr:fraisier des bois
```

Comment:
"fraisier des bois" is the name of the plant, and "fraise des bois" is the name of the fruit. If those were taken from wikidata / wikipedia, it's possible that some of the other languages list the plant and not the fruit.

Reply:
It's a Wikidata import indeed. I'll see if I can confirm some of them.

Reply:
I removed all the translations I couldn't find on OFF or otherwise wasn't quite sure about. There was actually [Wikidata item for the fruit](https://www.wikidata.org/wiki/Q104219130), but that wasn't much of a help.